documentation/openapi.json

```diff
@@ -985,6 +1012,24 @@
           }
         },
         "additionalProperties": false
+      },
+      "DotnetMonitorInfo": {
```

Comment:
You'll need to add documentation of this object to documentation\api\definitions.md